src/reloader.js

```diff
@@ -0,0 +1,7 @@
+import getData from './getData.js';
+
+export default function reloader() {
+  const scoreNode = document.querySelector('.score-list');
+  scoreNode.innerHTML = '';
+  getData();
+}
```

Comment:
- [ ] As I ask above USe arrow function. Please fix this minor issue. 
- [ ] Also check the `saveInput.js` and use arrow function. 

👍 💯

Reply:
I totally understand it's important to use arrow functions and that I should've used them. However using function keyword is not an issue. Also it's **not required** neither by the given JS best practices @ https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md nor by the project requirements 

"**Project requirements**
Read the Leaderboard API documentation to learn how to use this API.
Create a new game with the name of your choice by using the API.
Implement the "Refresh" button (receiving data from the API and parsing the JSON).
Implement the form "Submit" button (sending data to the API).
Use async and await JavaScript features to consume the API.
No styling is required."

With that said, thank you very much for reminding me of using them and for the valuable hints. I'll definitely do the changes tomorrow. Cheers :+1:

Reply:
I totally understand it's important to use arrow functions and that I should've used them. However using function keyword is not an issue. Also it's **not required** neither by the given JS best practices @ https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md nor by the project requirements 

"**Project requirements**
Read the Leaderboard API documentation to learn how to use this API.
Create a new game with the name of your choice by using the API.
Implement the "Refresh" button (receiving data from the API and parsing the JSON).
Implement the form "Submit" button (sending data to the API).
Use async and await JavaScript features to consume the API.
No styling is required."

With that said, thank you very much for reminding me of using them and for the valuable hints. I'll definitely do the changes tomorrow. Cheers :+1: